projects/1.20/assets/the-graveyard-forge/graveyard/lang/zh_cn.json

```
@@ -0,0 +1,311 @@
+  "block.graveyard.cracked_blackstone_gravestone": "裂纹磨制黑石砖墓碑",
+  "block.graveyard.stone_bricks_gravestone": "石砖墓碑",
+  "block.graveyard.mossy_stone_bricks_gravestone": "苔石砖墓碑",
+  "block.graveyard.bricks_gravestone": "红砖墓碑",
+  "block.graveyard.gilded_blackstone_gravestone": "镶金黑石墓碑",
+  "block.graveyard.sandstone_gravestone": "砂岩墓碑",
+  "block.graveyard.red_sandstone_gravestone": "红砂岩墓碑",
+  "block.graveyard.quartz_bricks_gravestone": "石英砖墓碑",
+
+  "item.graveyard.incarnated_evil_music_disc": "§b音乐唱片",
+  "item.graveyard.incarnated_evil_music_disc.desc": "Rotch Gwylt - Incarnated Evil",
+  "item.graveyard.bone_dagger": "骨头匕首",
+  "item.graveyard.dark_iron_ingot": "暗铁锭",
+  "item.graveyard.corruption": "堕落精华",
+  "item.graveyard.sarcophagus": "石棺",
+  "block.graveyard.sarcophagus": "石棺",
+  "item.graveyard.oak_coffin": "橡木棺材",
+  "block.graveyard.oak_coffin": "橡木棺材",
+  "item.graveyard.dark_oak_coffin": "深色橡木棺材",
+  "block.graveyard.dark_oak_coffin": "深色橡木棺材",
+  "item.graveyard.birch_coffin": "白桦木棺材",
+  "block.graveyard.birch_coffin": "白桦木棺材",
+  "item.graveyard.spruce_coffin": "云杉木棺材",
+  "block.graveyard.spruce_coffin": "云杉木棺材",
+  "item.graveyard.jungle_coffin": "丛林木棺材",
+  "block.graveyard.jungle_coffin": "丛林木棺材",
+  "item.graveyard.acacia_coffin": "金合欢木棺材",
+  "block.graveyard.acacia_coffin": "金合欢木棺材",
+  "item.graveyard.warped_coffin": "诡异木棺材",
+  "block.graveyard.warped_coffin": "诡异木棺材",
+  "item.graveyard.crimson_coffin": "绯红木棺材",
+  "block.graveyard.crimson_coffin": "绯红木棺材",
+  "item.graveyard.mangrove_coffin": "红树木棺材",
+  "block.graveyard.mangrove_coffin": "红树木棺材",
+  "item.graveyard.bamboo_coffin": "竹棺材",
+  "block.graveyard.bamboo_coffin": "竹棺材",
+  "item.graveyard.cherry_coffin": "樱花木棺材",
+  "block.graveyard.cherry_coffin": "樱花木棺材",
+
+  "item.graveyard.vial_of_blood": "小血瓶",
+  "block.graveyard.lower_bone_staff": "不祥的骨杖残片",
+  "block.graveyard.middle_bone_staff": "不祥的骨杖残片",
+  "block.graveyard.upper_bone_staff": "不祥的骨杖残片",
+  "block.graveyard.altar": "祭坛",
+  "block.graveyard.altar_side": "堕落深石",
+  "block.graveyard.altar_corner": "堕落深石边缘",
+  "block.graveyard.altar_center": "堕落深石中心",
+  "item.graveyard.white_bone_staff": "扎特拉克·古尔之杖",
+  "item.graveyard.black_bone_staff": "诺托克·古尔之杖",
+  "item.graveyard.red_bone_staff": "阿兹拉塔·古尔之杖",
+  "item.graveyard.cyan_bone_staff": "米纳克塔·古尔之杖",
+  "item.graveyard.purple_bone_staff": "夏普诺克·古尔之杖",
+  
+
+  "entity.graveyard.ghouling.wait": "我会待在这里。",
+  "entity.graveyard.ghouling.nowait": "我会跟着你。",
+  "entity.graveyard.ghouling.spawn": "我会服从你……",
+  "entity.graveyard.ghouling.respawn": "死亡……只不过是一点小小的挫折……",
+  "entity.graveyard.ghouling.obey": "我不会服从你的命令，你不是我的主人！",
+  "entity.graveyard.ghouling.kill": "我会让它停止行动！",
+  "entity.graveyard.nameless_hanged.wait": "我看到你了……不过再让我享受会阳光，晚上再回来见我吧。",
+
+  "subtitles.entity.graveyard.ghouling.spawn": "食尸灵：现身",
+  "subtitles.entity.graveyard.ghouling.death": "食尸灵：死亡",
+  "subtitles.entity.graveyard.ghouling.ambient": "食尸灵：呻吟",

```

```suggestion
  "subtitles.entity.graveyard.ghouling.ambient": "食尸灵：低吼",
```
是根据僵尸的字幕修改的，可以看下是否合适